backend/iam/adapter.py

```
@@ -10,13 +10,16 @@
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.db.models import Q
+from django.http import HttpResponseRedirect
 from django.utils.http import url_has_allowed_host_and_scheme
 from rest_framework.authtoken.serializers import AuthTokenSerializer
 from rest_framework.response import Response
 from rest_framework.status import HTTP_401_UNAUTHORIZED
 
 import structlog
 
+from iam.utils import generate_token
```

Ruff (0.11.9)

`iam.utils.generate_token` imported but unused

Remove unused import: `iam.utils.generate_token`

(F401)